data_collection_duration: 240 # seconds
num_of_iterations: 12 # number of iterations for training
num_of_epochs: 100 # number of epochs for training at each iteration
lr: 0.0005 # learning rate
weight_decay: 0.0 # weight decay
